package.json

```diff
@@ -14,6 +14,7 @@
     "prepublish": "yarn run build"
   },
   "dependencies": {
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
Not that big of a problem, but this should probably be under devDependencies.

Reply:
Yeah, I’ll fix.